### ISSUE TITLE
[Charge

### ISSUE THREAD

**Issue by xxdannilinxx** [woovibr/woovi-developers#423]

**Comment by daniloab** [comment]
Yes, it is correct because the fee is related to the pix receive service. The refund in our platform is free. But, you need to have amount necessary to refund it

**Comment by xxdannilinxx** [comment]
Maybe this should be rethought, you may have legal problems in the future with this.

You pay a fee to the bank even if the pix is ​​reversed or cancelled, is that it?

I have already integrated with several other players in the market and have never seen this.

Just to clarify!

**Comment by xxdannilinxx** [comment]
When I integrate the credit card with you and the customer chooses to cancel the purchase, won't the fee also be refunded?
